ConfigSpace/conditions.py

```diff
@@ -134,27 +136,28 @@ def __init__(self, *args):
                 raise ValueError("All Conjunctions and Conditions must have "
                                  "the same child.")
 
-    def get_descendant_literal_conditions(self):
-        children = []
+    def get_descendant_literal_conditions(self) -> List[ConditionComponent]:
```

Comment:
Wouldn't this only return AbstractCondition? If it is a conjunction, it is replaced by all conditions which make up the conjunction.